docs/source/getting_started/debugging.rst

```diff
@@ -0,0 +1,34 @@
+.. _debugging:
+
+Debugging Tips
+===============
+
+Debugging hang/crash issues
+-----------------------------
```

Comment:
```suggestion
---------------------------
```

I think normal practice is the line length matches the sentence

Reply:
fixed in [cb4d7ff](https://github.com/vllm-project/vllm/pull/5409/commits/cb4d7fff82d1bff7de12121bf96a04e4705895d0)